pkg/server/endpoints/endpoints.go

```diff
@@ -39,6 +39,8 @@ import (
 // route to the server in the case of a change in DNS membership.
 const defaultMaxConnectionAge = 3 * time.Minute
 
+const alpnProtoH2 = "h2"
```

Comment:
nit: Can this constant be referenced from [golang.org/x/net/http2](https://pkg.go.dev/golang.org/x/net/http2#pkg-constants)?